website/docs/d/policy_ipsec_vpn_local_endpoint.html.markdown

```diff
@@ -1,20 +1,20 @@
 ---
-subcategory: "Policy - Gateways and Routing"
+subcategory: "Beta"
 layout: "nsxt"
-page_title: "NSXT: policy_local_endpoint"
-description: Policy Local Endpoint data source.
+page_title: "NSXT: policy_ipsec_vpn_local_endpoint"
+description: Policy IPSec VPN Local Endpoint data source.
 ---
 
-# nsxt_policy_local_endpoint
+# nsxt_policy_ipsec_vpn_local_endpoint
 
-This data source provides information about policy local endpoint configured on NSX.
+This data source provides information about IPSec VPN policy local endpoint configured on NSX.
 
-This data source is applicable to NSX Global Manager, NSX Policy Manager and VMC.
+This data source is applicable to NSX Policy Manager and VMC.
 
 ## Example Usage
 
 ```hcl
```

Comment:
do we need to change this into terraform? same for d/policy_ipsec_vpn_service and d/l2_vpn_service

Reply:
we are using `hcl` in all object docs, I'm just trying `terraform` here in hope that it may fix formatting issues on registry doc website.